frontend/.eslintrc.json

```diff
@@ -1,62 +0,0 @@
-{
```

Comment:
This file was removed together with autogenerated package-lock.json because of some conflicts of ESlint's plugins versions in the package.json file (by restoring Schedule application locally on Intellij Idea). After resolving conflicts this file should be in the repository. So, removing of .eslintrc.json was commited accidentally.